scripts/zones/RuLude_Gardens/npcs/Magian_Moogle_Blue.lua

```diff
@@ -4,46 +4,40 @@
 -- Type: Magian Trials NPC (Relic Armor)
 -- !pos -6.843 2.459 121.9 64
 -----------------------------------
-require("scripts/globals/settings")
-require("scripts/globals/keyitems")
 require("scripts/globals/magiantrials")
+require("scripts/globals/status")
 -----------------------------------
 local entity = {}
+local EVENT_IDS = {
```

Comment:
I wonder if we could come to a common standard for this paradigm, though the PR I'm currently working on is based on zone, which makes things a bit easier to table up on the global side.

Reply:
I hesitated to put it in the IDs file of the zone however it only concerns a npc only. What do you want to do ?

Reply:
ok if I understand correctly, I make a table in magian trials allowing mapping npcids => eventsid?

Reply:
I'm just tossing ideas out.  I really like the idea of keeping everything in one place, but this is absolutely not necessary since your NPCs are nestled in one zone only.